cmd/miscellaneous.go

```diff
@@ -36,3 +37,7 @@ func printVersionInfo() {
 	logger.Infof("Go Version: %s", runtime.Version())
 	logger.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
 }
+
+func initializeValidatorFlags(cmd *cobra.Command) {
```

Comment:
Wouldn't it be better to keep this function within cmd/initializer.go?

Reply:
Done.